docs/chapter_sorting/radix_sort.md

```diff
@@ -0,0 +1,190 @@
+---
+comments: true
+---
+
+# 基数排序
+
+「基数排序 Radix Sort」是基于**分桶**思想实现的**非比较型**稳定的排序算法，是对**桶排序的扩展**。
```

Comment:
A little bit complicated.

"分桶" is not a "思想" because it is not a general concept.
"非比较型" and "稳定" are some characteristics that will be introduced in the article.